server/data/index.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/first */
+/*
+ * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
+ * In particular, applicationinsights automatically collects bunyan logs
+ */
+import { initialiseAppInsights, buildAppInsightsClient } from '../utils/azureAppInsights'
+import HmppsAuthClient from './hmppsAuthClient'
+import { createRedisClient } from './redisClient'
+import TokenStore from './tokenStore'
+
+initialiseAppInsights()
+buildAppInsightsClient()
+
+type RestClientBuilder<T> = (token: string) => T
+
+export const dataAccess = () => ({
+  hmppsAuthClient: new HmppsAuthClient(new TokenStore(createRedisClient({ legacyMode: false }))),
+})
+
+export type DataAccess = ReturnType<typeof dataAccess>
+
+export { HmppsAuthClient, RestClientBuilder }
```

Comment:
Redeclaring these tidies up imports a bit: 

You can mock out all data layer with: `jest.mock('../data')`
You can import multiple things in one line: `import { HmppsAuthClient, RestClientBuilder } from '../data'`